ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -971,7 +971,13 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </FlexRow>
           </WorkspaceEditSection>}
         <hr style={{marginTop: '1rem'}}/>
-        <WorkspaceEditSection header='Research Use Statement Questions' largeHeader={true}
+        <WorkspaceEditSection header={<FlexRow style={{alignItems: 'center'}}>
+          <div>Research Use Statement Questions</div>
+          <StyledAnchorTag href='https://aousupporthelp.zendesk.com/knowledge/articles/360042673211/en-us?brand_id=360001438652'
```

Comment:
The link doesn't work for me. Presumably it should work if we already have the article ID

Reply:
Its a saml problem. Is it taking you to a 403 error like this? 
<img width="763" alt="Screen Shot 2020-05-01 at 3 42 43 PM" src="https://user-images.githubusercontent.com/15351021/80836281-6a350f80-8bc2-11ea-913b-2bd11744a776.png">